.prettierignore

```diff
@@ -19,6 +16,5 @@ node_modules
 .parcel-cache
 
 # Test
-demo
```

Comment:
[dust] I don't really see a reason why we should ignore demo since it should automatically lint anyway 🤷‍♀️